fenix/app/src/main/res/values/strings.xml

```diff
@@ -610,6 +610,10 @@
     <!-- Error displayed when user attempts to install an add-on from AMO (addons.mozilla.org) that is already installed -->
     <string name="addon_already_installed">Add-on is already installed</string>
 
+    <!-- Add-on process crash dialog to user -->
+    <string name="addon_process_crash_dialog_title">Add-ons are temporarily disabled</string>
+    <string name="addon_process_crash_dialog_message">One or more add-ons stopped working, making your system unstable. Firefox unsuccessfully tried to restart the add-on(s).\n\nAdd-ons won’t be restarted during your current session.\n\nIf this continues to happen, try to remove or disable some add-ons.</string>
```

Comment:
We will need to a localization comment for both of these new strings describing what they're used for. This is to help provide context to our translators.

Reply:
Done!